packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx

```diff
@@ -33,92 +45,70 @@ describe('ColorPicker', () => {
-
-    const blueTableHeader = tableHeaders[3];
-    expect(blueTableHeader.textContent).toEqual(ColorPicker.defaultProps.blueLabel);
+    expect(alphaSlider.exists()).toBe(false);
+    expect(tableHeaders).toHaveLength(expectedTableComponents);
+    expect(tableInputs).toHaveLength(expectedTableComponents);
+  });
 
-    const alphaTableHeader = tableHeaders[4];
-    expect(alphaTableHeader.textContent).toEqual(ColorPicker.defaultProps.alphaLabel);
+  it('Renders default RGBA/Hex strings', () => {
+    const wrapper = mount(<ColorPicker color="#FFFFFF" />);
+
+    const tableHeaders = wrapper.find('.ms-ColorPicker-table > thead > tr > td');
+    const textHeaders = [
+      ColorPickerBase.defaultProps.hexLabel,
+      ColorPickerBase.defaultProps.redLabel,
+      ColorPickerBase.defaultProps.greenLabel,
+      ColorPickerBase.defaultProps.blueLabel,
+      ColorPickerBase.defaultProps.alphaLabel
+    ];
+
+    tableHeaders.forEach((node, index) => {
+      expect(node.text()).toEqual(textHeaders[index]);
+    });
   });
 
   it('Renders custom RGBA/Hex strings', () => {
-    const customHexLabel = 'Custom Hex';
-    const customRedLabel = 'Custom Red';
-    const customGreenLabel = 'Custom Green';
-    const customBlueLabel = 'Custom Blue';
-    const customAlphaLabel = 'Custom Alpha';
+    const textHeaders = ['Custom Hex', 'Custom Red', 'Custom Green', 'Custom Blue', 'Custom Alpha'];
 
-    const component = ReactTestUtils.renderIntoDocument(
+    const wrapper = mount(
       <ColorPicker
         color="#FFFFFF"
-        hexLabel={customHexLabel}
-        redLabel={customRedLabel}
-        greenLabel={customGreenLabel}
-        blueLabel={customBlueLabel}
-        alphaLabel={customAlphaLabel}
+        hexLabel={textHeaders[0]}
+        redLabel={textHeaders[1]}
+        greenLabel={textHeaders[2]}
+        blueLabel={textHeaders[3]}
+        alphaLabel={textHeaders[4]}
       />
-    ) as ColorPicker;
-
-    const renderedDOM = ReactDOM.findDOMNode(component as React.ReactInstance) as Element;
-    const tableHeaders = renderedDOM.querySelectorAll('.ms-ColorPicker-table > thead > tr > td') as NodeListOf<
-      HTMLTableDataCellElement
-    >;
-
-    const hexTableHeader = tableHeaders[0];
-    expect(hexTableHeader.textContent).toEqual(customHexLabel);
-
-    const redTableHeader = tableHeaders[1];
-    expect(redTableHeader.textContent).toEqual(customRedLabel);
-
-    const greenTableHeader = tableHeaders[2];
-    expect(greenTableHeader.textContent).toEqual(customGreenLabel);
-
-    const blueTableHeader = tableHeaders[3];
-    expect(blueTableHeader.textContent).toEqual(customBlueLabel);
+    );
 
-    const alphaTableHeader = tableHeaders[4];
-    expect(alphaTableHeader.textContent).toEqual(customAlphaLabel);
+    const tableHeaders = wrapper.find('.ms-ColorPicker-table > thead > tr > td');
+    tableHeaders.forEach((node, index) => {
+      expect(node.text()).toEqual(textHeaders[index]);
+    });
```

Comment:
We should probably start adding standardized tests for these conversions making use of `Customizer` with custom themes and custom class naming. I think if you did that here you would find a test failure until the styling ordering with the use of `className` is reordered. I think this would also be a good way to help prevent theming regression.